pkg/instrumentation/dotnet.go

```diff
@@ -28,17 +28,32 @@ const (
 	dotNetOTelAutoHomePath              = "/otel-auto-instrumentation-dotnet"
 	dotNetSharedStorePath               = "/otel-auto-instrumentation-dotnet/store"
 	dotNetStartupHookPath               = "/otel-auto-instrumentation-dotnet/net/OpenTelemetry.AutoInstrumentation.StartupHook.dll"
+	dotNetAutoPlugins                   = "AWS.Distro.OpenTelemetry.AutoInstrumentation.Plugin, AWS.Distro.OpenTelemetry.AutoInstrumentation"
 	dotnetVolumeName                    = volumeName + "-dotnet"
 	dotnetInitContainerName             = initContainerName + "-dotnet"
 	dotnetInstrMountPath                = "/otel-auto-instrumentation-dotnet"
 )
 
+const (
```

Comment:
@vastin  can you confirm these windows mount points too?

Reply:
Yes. It is confirmed.